cosmic/src/kick.f

```diff
@@ -87,6 +88,13 @@ SUBROUTINE kick(kw,m1,m1n,m2,ecc,sep,jorb,vk,snstar,
 * radius in solar units.
       gmrkm = 1.906125d+05
 
+* Set values for mean NS mass and mean ejecta as in Giacobbo & Mapelli 2020
```

Comment:
Is there any reason we might want to have these as variables that the user puts in? I'm most asking because I wonder if the values will change once we have more NS binary observations that change the merger rates?

Reply:
I thought about this, but I worry that this will introduce way too many flags. They're easy enough to change in the code directly, since they're set right at the beginning of kick.f. Thoughts?